src/encode.rs

```diff
@@ -155,6 +155,12 @@ impl Encode for &str {
     }
 }
 
+impl Encode for char {
```

Comment:
Should probably add an `is_empty` implementation (e.g. if the numeric value is equal to zero) together with a test in `encode_empty`.